src/main/scala/kamon/influxdb/InfluxDBClient.scala

```diff
@@ -18,29 +18,42 @@ package kamon.influxdb
 
 import java.net.InetSocketAddress
 
-import akka.actor.{ Actor, ActorContext, ActorRef, ActorSystem }
-import akka.event.Logging
-import akka.io.{ IO, Udp }
+import akka.actor._
+import akka.event.{ Logging, LoggingAdapter }
+import akka.io.Udp
 import akka.util.ByteString
-import com.typesafe.config.{ Config, ConfigFactory }
-import spray.can.Http
-import spray.client.pipelining._
-import spray.http._
-import spray.httpx.encoding.Deflate
+import com.typesafe.config.Config
+import org.asynchttpclient._
 
 import scala.concurrent.duration._
-import scala.util.{ Failure, Success }
+
+trait HttpClient {
+  def post(uri: String, payload: String): Unit
+}
+
+class AsyncHttpClient(logger: LoggingAdapter) extends HttpClient {
+  protected val aclient = new DefaultAsyncHttpClient()
+
+  override def post(uri: String, payload: String) =
+    aclient.preparePost(uri).setBody(payload).execute(
+      new AsyncCompletionHandler[Response] {
+        override def onCompleted(response: Response): Response = {
+          logger.info(s"${response.getStatusCode} ${response.getStatusText}")
```

Comment:
wdyt about changing this message to debug level? once all is working, this is just adding noise to the logs. Also something like "Metrics POST to InfluxDB completed with status code X - XTexr" as a message would be a lot easier to identify.